packages/nextjs/README.md

```
@@ -0,0 +1,93 @@
+That’s it! Since this is just handling key verification, there’s no need to specify a root key as an environment variable.
+
+If you want to customize how withUnkey processes incoming requests, you can do so as follows:
+
+## ```getKey```
+By default, withUnkey will look for a bearer token located in the ```authorization``` header. If you want to customize this, you can do so by passing a getter in the configuration object:
+
+```ts
+export const GET = withUnkey(async (req) => {
+  // ...
+}, {
+	getKey: (req) => new URL(req.url).searchParams.get("key"),
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)